### ISSUE TITLE
Uploading contract fails with `FailedToConvertParameter`

### ISSUE THREAD

**Issue by cmichi** [polkadot-js/apps#5121]
When I try to deploy ink!'s [flipper example](https://github.com/paritytech/ink/tree/master/examples/flipper) to our `canvas-node` I get this error:

```
contracts.instantiateWithCode
1002: Verification Error: Runtime error: Execution failed: 
ApiError(
  FailedToConvertParameter {
    function: "validate_transaction",
    parameter: "tx",
    error: Error {
      cause: None,
      desc: "Could not decode `MultiAddress`, variant doesn\'t exist"
    }
  }
): RuntimeApi,
Execution failed: ApiError,
FailedToConvertParameter {
  function: \"validate_transaction\",
  parameter: \"tx\",
  error: Error {
    cause: None,
    desc: \"Could not decode `MultiAddress`, variant doesn\\\'t exist\"
  }
}
```

The same issue appeared in the `canvas-ui`: https://github.com/paritytech/canvas-ui/issues/77. It still happens when I update our `canvas-node` to Substrate `master`.

I attached the `flipper.contract` to this issue, it can be used to reproduce the issue:
[flipper.contract.zip](https://github.com/polkadot-js/apps/files/6362880/flipper.contract.zip)

**Comment by jacogr** [comment]
See https://polkadot.js.org/docs/api/FAQ#i-cannot-send-transactions-sending-yields-decoding-failures

Here is the type definitions for canvas, it is fixed to the older address formats - https://github.com/polkadot-js/apps/blob/master/packages/apps-config/src/api/spec/canvas.ts#L14-L18

TL;DR Based on the spec_version where the address changes, the types are to be updated

**Comment by jacogr** [comment]
Would appreciate a ping or PR from the team when they do change types, like they have done here to ensure stuff continues working. Attached PR should be deployed in 15 mins (depending on CI).
